LEAF_Request_Portal/admin/templates/view_form_library.tpl

```diff
@@ -135,16 +136,17 @@ $(function() {
+    <span style="position: absolute; color: transparent" aria-atomic="true" aria-live="assertive" id="filterStatus" role="status"></span>
+    <a role="button" class="buttonNorm" tabindex="0" href="?a=form" style="display: inherit; width: 150px; text-decoration: none; color: black" id="backToForm"><img src="../../libs/dynicons/?img=edit-undo.svg&amp;w=32" alt="Go back" title="Go back"/> Back to My Forms</a>
     <br /><br />
     Filter by Business Lines:
-    <div class="buttonNorm" onclick="applyFilter('');" style="width: 150px"><img src="../../libs/dynicons/?img=Accessories-dictionary.svg&amp;w=32" alt="Icon" title="Icon" /> All Business Lines</div>
+    <div role="button" onkeydown="triggerKeydown(event, this.id)" class="buttonNorm" tabindex="0" onclick="applyFilter('')" style="width: 150px" id="all_Business_Lines"><img aria-hidden="true" src="../../libs/dynicons/?img=Accessories-dictionary.svg&amp;w=32" alt="Icon" title="Icon"/> All Business Lines</div>
     <br />
-    <div class="buttonNorm" onclick="applyFilter('Administrative');" style="width: 150px"><img src="../../libs/dynicons/?img=applications-office.svg&amp;w=32" alt="Icon" title="Icon" /> Administrative</div>
-    <div class="buttonNorm" onclick="applyFilter('Human Resources');" style="width: 150px"><img src="../../libs/dynicons/?img=system-users.svg&amp;w=32" alt="Icon" title="Icon" /> Human Resources</div>
-    <div class="buttonNorm" onclick="applyFilter('Information Technology');" style="width: 150px"><img src="../../libs/dynicons/?img=network-idle.svg&amp;w=32" alt="Icon" title="Icon" /> Information Technology</div>
-    <div class="buttonNorm" onclick="applyFilter('Logistics');" style="width: 150px"><img src="../../libs/dynicons/?img=package-x-generic.svg&amp;w=32" alt="Icon" title="Icon" /> Logistics</div>
-    <div class="buttonNorm" onclick="applyFilter('Fiscal');" style="width: 150px"><img src="../../libs/dynicons/?img=x-office-spreadsheet.svg&amp;w=32" alt="Icon" title="Icon" /> Fiscal</div>
+    <div role="button" onkeydown="triggerKeydown(event, this.id)" class="buttonNorm" tabindex="0" onclick="applyFilter('Administrative')" style="width: 150px" id="Administrative"><img aria-hidden="true" src="../../libs/dynicons/?img=applications-office.svg&amp;w=32" alt="Icon" title="Icon" /> Administrative</div>
+    <div role="button" onkeydown="triggerKeydown(event, this.id)" class="buttonNorm" tabindex="0" onclick="applyFilter('Human Resources')" style="width: 150px" id="Human Resources"><img aria-hidden="true" src="../../libs/dynicons/?img=system-users.svg&amp;w=32" alt="Icon" title="Icon" /> Human Resources</div>
+    <div role="button" onkeydown="triggerKeydown(event, this.id)" class="buttonNorm" tabindex="0" onclick="applyFilter('Information Technology')" style="width: 150px" id="Information Technology"><img aria-hidden="true" src="../../libs/dynicons/?img=network-idle.svg&amp;w=32" alt="Icon" title="Icon" /> Information Technology</div>
```

Comment:
ID elements cannot contain spaces. 

Unless the id elements are needed for the screen reader, my suggestion would be to remove the ids all together and just pass `this` as the second parameter to triggerkeydown(). `this` in this case would refer to the whole element triggering the onClick.

Reply:
Addressed in commit 0c0